### PR TITLE
Add initial getting started & contributing documentation

### LINKED ISSUE #8: Create Getting Started Documentation to Onboard Additional Collaborators
## Problem/Opportunity Statement
There's no documentation to show how someone would spin up a development instance of Hubcast in order to help contribute to the project. This presents an immediate barrier to both adoption and collaboration.

## What would success / a fix look like?
- A set of docs describing how to configure GitHub, GitLab, and Hubcast to get started mirroring a project.
- Additionally we should show how you can use smee.io to develop Hubcast locally without access to a public IP.

### PR DESCRIPTION
**Pull request by alecbcs**
Closes #8. This PR adds an initial set of docs to help collaborators more easily get an instance of Hubcast setup with a testing repository.

### CODE REVIEW

**Comment by alecbcs** [comment]
@tgmachina @cook107 @becker33 here's an initial set of getting started docs. If you all could take a look at these sometime this week, I'd welcome any feedback you all might have. I think there's still a lot of room for improvement but I think this helps folks get a start on the project.

**Comment by tgmachina** [comment]
@alecbcs, LGTM! While I was reading the setup docs, I thought about making some of the test setup for local dev easier and opened #13.

File: docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+After cloning the repository you'll need to follow the [Getting Started](getting-started.md)
+documentation to setup a testing,
+1. GitHub Repo
+2. GitHub App
+3. GitLab Repo
+4. GitLab Repo Webhook
+5. GitLab Repo Access Token
+
+> [!TIP]
+> If you're developing locally you can use [smee.io](https://smee.io) to relay
```
Comment (tgmachina) [inline docs/contributing.md:30]: Should we add `smee` as [an optional dependency](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#dependencies-optional-dependencies)?

**Review by tgmachina — APPROVED** [review]

File: docs/contributing.md (hunk 1 above)
Comment (alecbcs) [reply docs/contributing.md:30]: Haha no worries 😂

**Pull request merged**